### PR TITLE
fix(cli): params are strings

### PR DESCRIPTION
**Pull request by heafalan**
Since the id type is string, there is a problem for updating or deleting an entry via the cli. This could be fix it.

### CODE REVIEW

File: packages/cli/main.js [hunk 1]
```diff
@@ -4,7 +4,10 @@ import hrp from "http-request-plus";
 import { format, parse } from "json-rpc-protocol";
 
 async function main() {
-  const params = require("minimist")(process.argv.slice(2));
+  const getopts = require("getopts");
+  const params = getopts(process.argv.slice(2), {
+    string: ["id"],
```
Comment (julien-f) [inline packages/cli/main.js]: Are you sure this is necessary?

The CLI should not have any knowledge regarding methods and their params.

Comment (heafalan) [reply packages/cli/main.js]: As the package `minimist` does, `getopts` takes `id` as a number.
But I can try to find another solution.

Comment (badrAZ) [reply packages/cli/main.js]: IMHO, it should be parsed in the method.

Comment (julien-f) [reply packages/cli/main.js]: IMHO, we should let minimist/getopts parses the params but we need to work-around the duck-typing behavior and manually convert all parsed params to strings.

**Review by badrAZ — APPROVED** [review]

**Comment by badrAZ** [comment]
This PR also fix the issue #38 .

**Pull request merged**